database/setup.js

```diff
@@ -0,0 +1,19 @@
+const db = require('./dbconfig');
+
+const setupTable = async () => {
+
+    const dropTable = `DROP TABLE IF EXISTS todo;`
+
+    const createTable = 
+    `CREATE TABLE IF NOT EXISTS todo(
+        id Serial PRIMARY KEY,
+        title VARCHAR (50) UNIQUE NOT NULL,
+        is_done BOOLEAN
+    );`
+
+    await db.raw(dropTable)
+    await db.raw(createTable)
```

Comment:
i guess this is to make sure that there is no data in the database before we create our table, right?

Reply:
Yes, and only meant for development purpose of course